docs/reference/aggregations/bucket/range-aggregation.asciidoc

```diff
@@ -379,30 +379,3 @@ Response:
 }
 --------------------------------------------------
 // TESTRESPONSE[s/\.\.\.//]
-
-If a sub aggregation is also based on the same value source as the range aggregation (like the `stats` aggregation in the example above) it is possible to leave out the value source definition for it. The following will return the same response as above:
```

Comment:
This doc change seems unrelated to this PR which is about adding infrastructure?